sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -548,9 +548,8 @@ MarginCalculationLauncher::launchScenario(const MultiVariantInputs& inputs, cons
     subModels.insert(subModels.end(), subModelsToAdd.begin(), subModelsToAdd.end());
     subModelsToAdd = modelMulti->findSubModelByLib(DDBDir + "/TfoTrippingEvent" + DYN::sharedLibraryExtension());
     subModels.insert(subModels.end(), subModelsToAdd.begin(), subModelsToAdd.end());
-    subModelsToAdd = modelMulti->findSubModelByLib(DDBDir + "/EventConnectedStatus" + DYN::sharedLibraryExtension());
-    subModels.insert(subModels.end(), subModelsToAdd.begin(), subModelsToAdd.end());
     subModelsToAdd = modelMulti->findSubModelByLib(DDBDir + "/EventQuadripoleConnection" + DYN::sharedLibraryExtension());
```

Comment:
This one is also added twice no?

Reply:
it's Disconnection no ? I see it only once